swimster-api/models/Reservation.js

```diff
@@ -0,0 +1,195 @@
+const db = require('../db')
+const { BadRequestError } = require('../utils/error')
+
+class Reservation {
+    static get TAX_RATE() {
+        return 0.0725
+    }
+
+    static get FEES_RATE() {
+        return 0.08
```

Comment:
might be good to put a comment here as to why this is the fees rate as well as the tax rate above

Reply:
done!